### PR TITLE
Update API with latest swagger

### PR DESCRIPTION
**Pull request by waysidekoi**
This PR drops the existing `User` related models and controllers, and updates the `Appointment` model with the expected v1 attributes. Also adds a `Search` model to handle `search_id`s from a member or a personal assistant.

### CODE REVIEW

File: spec/requests/api_spec.rb [hunk 1]
```diff
@@ -2,136 +2,101 @@
...
+    end
+
+    describe "UPDATE /appointments" do
+      specify do
+        expect(subject).to validate(
+          :put, "/appointments/{id}", 200, { 'id' => appointment_id }.merge('_data' => appointment_attributes)
+        )
+      end
+
+      context "when member id is missing" do
```
Comment (rall) [inline spec/requests/api_spec.rb]: I guess `context 'foo' do specify do` is better expressed as `it 'foo' do`

File: app/models/boolean_adapter.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class BooleanAdapter
```
Comment (rall) [inline app/models/boolean_adapter.rb]: can we use dynamoid's :boolean adapter instead?

File: lib/tasks/db.rake [hunk 3]
```diff
@@ -32,8 +32,9 @@ namespace :db do
 
     count = [args[:count].to_i, 1].max
     count.times do
-      FactoryGirl.create(:appointment)
-      FactoryGirl.create(:user)
+      FactoryGirl.build(:appointment).tap { |appointment|
```
Comment (rall) [inline lib/tasks/db.rake]: for me the rule about blocks is ''use `do end` for multiline blocks unless part for a chain"

**Comment by rall** [comment]
some minor comments and a question about booleans

**Comment by rall** [comment]
you'll need to update the appointment model too I think

**Comment by waysidekoi** [comment]
Yup, had it the update for the `Appointment` model locally, but not pushed, oops

**Comment by rall** [comment]
👍

**Pull request merged**